components/Common/MetaBar/index.tsx

```diff
@@ -0,0 +1,63 @@
+import type { Heading } from '@vcarl/remark-headings';
+import { Fragment, useMemo } from 'react';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type MetaBarProps = {
+  items: Record<string, React.ReactNode>;
+  headings?: {
+    items: Heading[];
+    depth?: number;
+  };
+};
+
+// The default depth of headings to display in the table of contents.
+const HEADING_DEPTH: number = 2;
```

Comment:
No need to make a constant if we are only using it once, I guess you can simply to `depth = 2` on line 22

And add an inline comment of

`// This is the default depth we want for the Table of Contents`